pkg/scheduler/core/generic_scheduler.go

```diff
@@ -94,20 +95,20 @@ func (g *genericScheduler) Schedule(
 	}
 	klog.V(4).Infof("Feasible clusters scores: %v", clustersScore)
 
-	clusters, err := g.selectClusters(clustersScore, spec.Placement, spec)
+	selectedClusters, err := g.selectClusters(clustersScore, spec.Placement, spec)
 	if err != nil {
 		return result, fmt.Errorf("failed to select clusters: %w", err)
 	}
-	klog.V(4).Infof("Selected clusters: %v", clusters)
+	klog.V(4).Infof("Selected clusters: %v", selectedClusters)
```

Comment:
The type of `selectedCluster` is now different from the `clusters`, so the log should be updated accordingly. For instance, with this change, the log now looks like:
```
Selected clusters: [{member1 100 9 member1 9}]
```
It makes the log message hard to understand.

It dumps the slice of `ClusterDetailInfo` type:
- Name(`member1`)
- Score(`100`)
- AvailableReplicas (`9`)
- Cluster(`member1`)
- AllocatableReplicas(`9`)

Reply:
Found this when testing PR https://github.com/karmada-io/karmada/pull/6857.

Reply:
Sent #6880 for tracking this.